deployments/taiko_test/solcInputs/9671d4e1820229dcd69b349a99feb65b.json

```diff
@@ -62,4 +62,4 @@
       "useLiteralContent": true
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Was this manually added? If so, maybe we should leave it out as I believe this is a generated file, so we shouldn't modify it.

Reply:
I believe this is a formatting thing from prettier. I haven't touched the file myself.
Will remove this change from the commit entirely